packages/wasm/crate/src/metadata.rs

```diff
@@ -73,21 +83,32 @@ mod test_helpers {
 
     use super::*;
 
-    pub fn get_metadata_for(display_denom: &str) -> Metadata {
+    pub fn get_metadata_for(display_denom: &str, base_denom_is_display_denom: bool) -> Metadata {
```

Comment:
This is just a test helper. Auction NFTs only have one denom unit, so the base denom is the same as the display denom. So I added this arg to this function so that I can generate the right type of metadata for auction NFTs as well.

Reply:
Yeah — the display denom is the one you see in the UI (e.g., "penumbra") but the base denom is the smallest possible unit (e.g., "upenumbra"). Usually they're different, just like US dollars and cents are different, but auction NFTs apparently only have a single denom unit — which makes sense, since you'll never hold more than 1 NFT for a given auction.